.github/workflows/triton-benchmarks.yml

```
@@ -19,6 +19,10 @@ on:
           - ELAPSED_TIME
           - UPSTREAM_PYTORCH_PROFILER
         default: UPSTREAM_PYTORCH_PROFILER
+      no_verify:
+        description: Skip verification of the benchmark results
+        type: boolean
+        default: false
```

Maybe to simplify? (we can avoid double negatives)

```suggestion
      verify:
        description: Skip verification of the benchmark results
        type: boolean
        default: true
```

---

done